hibernate-core/src/main/java/org/hibernate/type/descriptor/jdbc/JdbcType.java

```diff
@@ -64,9 +76,10 @@ default String getFriendlyName() {
 	 * to obtain the SQL column type.
 	 *
 	 * @return a JDBC type code
+	 * @since 6.2
 	 */
-	default int getDefaultSqlTypeCode() {
-		return getJdbcTypeCode();
+	default int getDdlTypeCode() {
```

Comment:
I think the changes in this class should be reverted. The use of `getDdlTypeCode()` in `Column` should be substituted with `getDefaultSqlTypeCode()` for the 6.1 branch